components/ManualPreview/ManualPreview.module.css

```diff
@@ -0,0 +1,132 @@
+.manual {
+    position: relative;
+    background: white;
+    overflow: visible;
+    width: 90vw;
+    display: block;
+    cursor: pointer;
+    margin: 0;
+    border-radius: 0.5em;
+    aspect-ratio: 16 / 10;
+    margin-bottom: 1em;
+    text-decoration: none;
+    font-weight: 500;
+    font-size: clamp(18px, 5vw, 22px);
+}
+
+.manual:focus-visible {
+    outline-width: 0;
+}
+
+.manualEmoji {
+    position: absolute;
+    right: .1em;
+    bottom: .1em;
+    z-index: -1;
+    width: 100%;
+    height: 100%;
+}
+
+.manualInner {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    transform-origin: left bottom;
+    transition: transform .15s ease-in;
+    border-radius: inherit;
+    background-size: cover;
+    background-repeat: none;
+    background-position: left bottom;
+    padding: 1.2em;
+}
+
+.manualInner:before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    opacity: .175;
+    filter: brightness(1.25);
+    pointer-events: none;
+    border-radius: inherit;
+    background: currentColor;
+}
+
+.manualIcon {
+    position: absolute;
+    width: 1.75em;
+    height: 1.75em;
+    left: 1.25em;
+    bottom: 1em;
+}
+
+.manual:hover .manualInner {
+    transform: matrix(1, -0.08, 0, 1, 0, 0) scaleX(0.95);
+}
+
+.manual:focus-visible {
+    outline: 2px solid black;
+}
+
+.manualTitle {
+    position: relative;
+    font-size: 1em;
+    margin-bottom: .5em;
+}
+
+.manualSubtitle {
+    font-size: .55em;
+}
+
+.manualStatus {
+    position: absolute;
+    top: -.8vw;
+    right: -.8vw;
+    font-size: .9em;
+    padding: .25em .5em .1em;
+    line-height: 1;
+    color: white;
+    border-radius: 20px;
+    font-weight: 500;
+}
+
+.manualStatusNew {
+    background: #00b4ff;
+}
+
+.manualStatusUpdated {
+    background: #ffd400;
```

Comment:
Maybe move colors to css variable (so it will be easier to understand what kind of color it is 🙂)

Reply:
CSS variables with colors are needed to avoid duplication. So far this value is applied only once.

> will be easier to understand what kind of color 
I think
a variable with a color is unlikely to help you understand what kind of color it is and why it is needed :)